### PR TITLE
Config/tailwind

### LINKED ISSUE #6: Tailwind Configuration
Add the tailwind configuration according to the design system provided

### PR DESCRIPTION
**Pull request by jrTilak**
Fixes #6

- [x] Added Fonts
- [x] Added `xs` screen
- [x] Added Colors
- [x] Add tailwind config

### If any colors are missed, we can add it progressively
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        'primary/10': '#246BFD1A',
+        background: '#0C0C0D',
+        neutral: '#F4F4F4',
+        'neutral/10': '#F5F8FF',
+        'neutral/40': '#ABAEB2',
+        'neutral/60': '#7A7C80',
+        'neutral/90': '#313233',
+        'neutral/95': '#252526',
+        'neutral/100': '#141414',
+        yellow: '#FF9300',
```
Comment (aakashk1905) [inline tailwind.config.js:24]: Change yellow to coral

**Review by aakashk1905 — CHANGES_REQUESTED** [review]
make the changes

**Comment by jrTilak** [comment]
Done the changes.

**Review by aakashk1905 — APPROVED** [review]
LGTM

**Pull request merged**